data/benefits/javascript/haut-de-france-aide-transports-salaries.yml

```
@@ -19,9 +21,7 @@ profils:
     conditions: []
   - type: apprenti
     conditions: []
-link: https://guide-aides.hautsdefrance.fr/spip.php?page=dispositif&id_dispositif=458
-teleservice: https://aides.hautsdefrance.fr/sub/tiers/aides/details/?sigle=ATPS23
-instructions: https://guide-aides.hautsdefrance.fr/spip.php?page=dispositif&id_dispositif=458
+link: https://aides.hautsdefrance.fr/sub/tiers/aides/details/?sigle=ATPS23
```

```suggestion
link: https://aides.hautsdefrance.fr/sub/tiers/aides/details/?sigle=ATPS24
```
Sinon c'était l'aide de 2023.

---

J'ai aussi l'impression qu'il faut se créer un compte pour pouvoir faire une demande, ce n'est pas bien explicite : https://aides.hautsdefrance.fr/sub/tiers/authentification non ?